setup.py

```diff
@@ -2,7 +2,7 @@
 
 setup(
     name='edx-opaque-keys',
-    version='0.3.2',  # Please update CHANGELOG when you bump versions
+    version='0.3.1',
```

Comment:
But - perhaps we should just add any additional 0.3.3 changes into this PR to avoid confusion?